Kostyukevich/lab4.cpp

```diff
@@ -0,0 +1,93 @@
+#include "windows.h"
+#include "TlHelp32.h"
+#include <fstream>
+#include <iostream>
+#include <string>
+using namespace std;
+
+int main() {
+
+
+	ifstream deadListFile("deadList.txt");
+	ofstream logFile("log.txt");
+
+	TCHAR processName[200][MAX_PATH];
```

Comment:
You could use something like `std::vector<std::wstring>` or `std::vector<std::string>` since you are using C++.